From 5f8ac4f156d5d5dcf247b38e104f72eaa00e3d73 Mon Sep 17 00:00:00 2001
From: Herman van Hazendonk <github.com@herrie.org>
Date: Mon, 30 May 2022 15:23:38 +0200
Subject: [PATCH] main.qml: Fix syntax error

May 30 13:18:45 qemux86-64 sh[796]: WARNING: file:///usr/lib/maliit/plugins/qml/view-global/main.qml:348:10: Expected token `,'
May 30 13:18:45 qemux86-64 sh[796]:         toast popup

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>
---
Upstream-Status: Pending

 maliit-plugin-global/view/main.qml | 11 ++++++++---
 1 file changed, 8 insertions(+), 3 deletions(-)

diff --git a/maliit-plugin-global/view/main.qml b/maliit-plugin-global/view/main.qml
index 87c4a6a..4be5110 100755
--- a/maliit-plugin-global/view/main.qml
+++ b/maliit-plugin-global/view/main.qml
@@ -343,9 +343,14 @@ Rectangle {
             root.reset();
         }
         function onTranslatorChanged(speakLanguage) {
-            style.setFontForMenuLocale(PluginProxy.translatorLocale)
-            if (speakLanguage == false) return;
-            if (inputSource == 0 || visible == false) return; // because reading lang name through toast popup
+            style.setFontForMenuLocale(PluginProxy.translatorLocale);
+            if (speakLanguage === false) {
+                return;
+            }
+            if (inputSource == 0 || visible == false) { 
+                // because reading lang name through toast popup
+                return; 
+            }            
             ttsService.speakButton(globalStringModel.getStringLangCode(langCode));
         }
         function onCursorVisibleChanged() {
